x-pack/plugins/apm/public/hooks/use_filters_for_embeddable_charts.ts

```diff
@@ -6,27 +6,26 @@
  */
 
 import { useMemo } from 'react';
-import { SERVICE_NAME, TRANSACTION_TYPE } from '../../common/es_fields/apm';
+import { SERVICE_NAME } from '../../common/es_fields/apm';
 import { termQuery } from '../../common/utils/term_query';
 import { useApmParams } from './use_apm_params';
 import { environmentQuery } from '../../common/utils/environment_query';
 
 export function useFiltersForEmbeddableCharts() {
```

Comment:
I wonder if this hook is really necessary, I see that it is used in one file only. Maybe you could just move this to inside `public/components/app/mobile/service_overview/index.tsx`?

Reply:
The plan is to add the map to the transaction page as well. So it might be worth having the hook, no?